README.md

```diff
@@ -202,9 +202,9 @@ docker run --rm -v "${PWD}:/home/repo" returntocorp/semgrep --config r2c
 
 ## Contribution
 
-`semgrep` is LGPL-licensed, feel free to help out: [CONTRIBUTING](https://github.com/returntocorp/sgrep/blob/develop/CONTRIBUTING.md).
+`semgrep` is LGPL-licensed, feel free to help out: [CONTRIBUTING](https://github.com/returntocorp/semgrep/blob/develop/CONTRIBUTING.md).
 
-`semgrep` is a frontend to a larger program analysis library named [pfff](https://github.com/returntocorp/pfff/), where it was named `sgrep`. pfff began and was open-sourced at [Facebook](https://github.com/facebookarchive/pfff) but is now archived and the primary maintainer works at [r2c](https://r2c.dev).
```

Comment:
I'm not sure if we're trying to distinguish between `semgrep` and `sgrep` here. It seems like the 2nd portion of the sentence is mostly restating the beginning.

Reply:
The intention is to explain that the original pfff tool was named sgrep, for those looking to understand where that tool went.

Reply:
(I think there's some value to preserving that link sgrep -> semgrep FWIW)

Reply:
Sure thing, I'll add it back in with some context on the `sgrep` -> `semgrep` transition